packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -13,20 +13,31 @@ function findExampleByKey(httpContent: IHttpContent, exampleKey: string) {
   return httpContent.examples && httpContent.examples.find(example => example.key === exampleKey);
 }
 
+function hasContents(v: IHttpOperationResponse): v is PickRequired<IHttpOperationResponse, 'contents'> {
+  return !!v.contents;
+}
+
 function findBestHttpContentByMediaType(
-  response: IHttpOperationResponse,
+  response: PickRequired<IHttpOperationResponse, 'contents'>,
   mediaType: string[],
 ): IMediaTypeContent | undefined {
-  return response.contents.find(content =>
-    accepts({
-      headers: {
-        accept: mediaType.join(','),
-      },
-    }).type(content.mediaType),
-  );
+  return response.contents
+    .map(t => {
+      if (t.mediaType === '*') t.mediaType = '*/*';
```

Comment:
@chris-miaskowski Do you know/recall why the default media type has been marked with `*` instead of `*/*`? If there's no particular reason I could probably fix this in graphite and avoid this remap.

Reply:
Hey @XVincentX, no reason - was picked arbitrarily.

Reply:
Ah ok nice — then maybe we can make it happen with https://github.com/stoplightio/graphite/pull/191 and remove this?